src/adagrams.js

```diff
@@ -1,15 +1,150 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let playersHand = [];
+  const letterPool = getLetterPool(LETTER_POOL);
+
+  while (playersHand.length !== 10) {
+    let letter = randomLetter(letterPool);
```

Comment:
The `randomLetter` function mutates `letterPool`, which is a bit hard to follow.